src/commcare_cloud/terraform/modules/server/main.tf

```diff
@@ -16,7 +16,7 @@ resource aws_instance "server" {
     delete_on_termination = true
   }
   lifecycle {
-    ignore_changes = ["user_data", "key_name", "root_block_device.0.delete_on_termination", "ebs_optimized"]
+    ignore_changes = ["user_data", "key_name", "root_block_device.0.delete_on_termination", "ebs_optimized", "ami"]
```

Comment:
What does this do?

Reply:
Check out the commit message here: https://github.com/dimagi/commcare-cloud/pull/2424/commits/90b917dde46b809c4ae70a78be4bed843ec3a8a3

Adding a property to `ignore_changes` tells terraform that if that property on an existing resource differs from the desired value, just ignore it rather than destroying and recreating the resource. Here it means, don't destroy all of our current machines just because they were built from a different AMI.

Reply:
makes sense, thanks